modules/Bio/EnsEMBL/Compara/PipeConfig/ncRNAtrees_conf.pm

```diff
@@ -613,9 +632,8 @@ sub core_pipeline_analyses {
                                      },
               -flow_into          => [ 'write_stn_tags',
                                        # 'backbone_fire_homology_dumps',
-                                        WHEN('#do_cafe# and  #binary_species_tree_input_file#', 'CAFE_species_tree'),
-                                        WHEN('#do_cafe# and !#binary_species_tree_input_file#', 'make_full_species_tree'),
-                                    ],
+                                       WHEN('#do_cafe# and  #binary_species_tree_input_file#', 'CAFE_species_tree'),
+                                       WHEN('#do_cafe# and !#binary_species_tree_input_file#', 'make_full_species_tree'),],
```

Comment:
We need to put these lines back as they were to avoid/prevent git blame

Reply:
I will make these changes and initialize a database to see if it works.

Reply:
I have added the '-wait_for' directive, so the relevant portion of the pipeline now looks like this:

![image](https://user-images.githubusercontent.com/103430062/167158314-df8a18bf-8d3b-4236-8dab-9faab17a3f16.png)

I have tried to format the WHEN lines as suggested but git blame still shows me as author.